packages/Python/lldbsuite/test/lang/swift/clangimporter/config_macros/Makefile.dylib

```diff
@@ -0,0 +1,8 @@
+LEVEL = ../../../../make
```

Comment:
most tests call this file `dylib.mk`. We generally use .mk as a suffix for sub-makefiles (though they aren't technically makefile fragments, but proper standalone recursive files).